sdk/core/Azure.Core/src/Shared/DiagnosticScope.cs

```diff
@@ -111,7 +111,9 @@ private class DiagnosticActivity : Activity
         {
             private List<Activity>? _links;
 
-            public IEnumerable<Activity> Links => (IEnumerable<Activity>?)_links ?? Array.Empty<Activity>();
+#pragma warning disable 109
```

Comment:
Not: helpful for reviews if you keep the corresponding comment that VS would've added.